packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -171,7 +171,7 @@ public void onLayoutChange(
     if ((currentHeight != oldHeight || currentWidth != oldWidth)) {
       ReadableArray transformOrigin = (ReadableArray) v.getTag(R.id.transform_origin);
       ReadableArray transformMatrix = (ReadableArray) v.getTag(R.id.transform);
-      if (transformMatrix != null && transformOrigin != null) {
+      if (transformMatrix != null || transformOrigin != null) {
```

Comment:
This isn't a matrix most of the time, right? Can we rename?

Reply:
Agree, renamed to `transforms` since it's an array.